first/frc/index.md

```diff
@@ -13,3 +13,11 @@ This season's game is *FIRST* Infinite Recharge. The teaser and game rules are p
   <a class="btn" href="https://firstfrc.blob.core.windows.net/frc2020/Manual/2020FRCGameSeasonManual.pdf" target="_blank">Download Manual</a>
   <a class="btn" href="http://www.firstinspires.org/resource-library/frc/competition-manual-qa-system?utm_source=hs_email&utm_medium=email&utm_content=40176757&_hsenc=p2ANqtz-9sRtvBregjMA_3wwl7OVcCXZdE7pCduMWXk9_UYkxStV180YhEK_6jamp_zNM9VP7Dety_tN46Pex47KTi8U5wjLgX3Q&_hsmi=40176757" target="_blank">Other Resources</a>
 </div>
+
```

Comment:
I think if the page is restructured based on this comment, it will make it easer to restructure and rephrase the paragraphs.

Reply:
GitHub is messing with me -_- but okay. My comment:
How about it will go in this order: FIRST Robotics Competition, how does it work, fun facts, and then information about Infinite Recharge?

Reply:
I think that makes sense! Make sure to push any commits you make to 178Laptop and not team178 so that your new changes appear in this pull request.

Reply:
Okay, after reading all paragraphs, it seems that the paragraph "How does it work" and facts are much more general than the FIRST Robotics Competition one. FRC paragraph talks more specifically to our team. How about this order: How does it work, facts, FRC paragraph, and Infinite Recharge and just rephrase most of the context so it will look good together?